### PR TITLE
Np add fragment file task

### PR DESCRIPTION
**Pull request by nikellepetrillo**
### Description

We need to create a task that creates the fragment file for the multiome pipeline. 
----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [x] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

**Comment by nikellepetrillo** [comment]
job manager link --> https://job-manager.dsde-prod.broadinstitute.org/jobs/429fce4b-34de-43f8-9841-5b03e4719f0a

this job ran the atac wdl that is currently on this np_add_frag_file_task branch

File: beta-pipelines/multiome/atac.wdl [hunk 1]
```diff
@@ -255,3 +264,52 @@ workflow ATAC {
...
+    set -e pipefail
+
+    python3 <<CODE
+
+
+    barcodes_in_read_name = "~{barcodes_in_read_name}"
+    bam = "~{bam}"
+    bam_base_name = "~{bam_base_name}"
+
+    # if barcodes are in the read name, then use barcode_regex to extract them. otherwise, use barcode_tag
```
Comment (jessicaway) [inline beta-pipelines/multiome/atac.wdl:293]: do we already have a barcode tag?? There's a separate ticket to add one...

**Review by jessicaway — APPROVED** [review]
Minor comment and question, but looks great!

File: beta-pipelines/multiome/atac.wdl [hunk 2]
```diff
@@ -53,11 +55,18 @@ workflow ATAC {
...
+  call CreateFragmentFile {
+    input:
+      bam = BWAPairedEndAlignment.bam_aligned_output,
+      barcodes_in_read_name = barcodes_in_read_name,
   }
     
-  output {
-    File bam_aligned_output = BWAPairedEndAlignment.bam_aligned_output
-  }   
+    output {
```
[the remark this answers is not in the document]
Comment (nikellepetrillo) [reply beta-pipelines/multiome/atac.wdl]: good eye, sometimes my linter does weird things i dont catch it! fixed 👍

**Comment by aawdeh** [comment]
Looks good to me!

**Review by aawdeh — APPROVED** [review]

**Pull request merged**